test/e2e_test.go

```diff
@@ -195,8 +195,6 @@ var _ = Describe("MetricsServer", func() {
 			resp := mustProxyContainerProbe(restConfig, pod.Namespace, pod.Name, pod.Spec.Containers[0], pod.Spec.Containers[0].ReadinessProbe)
 			diff := cmp.Diff(string(resp), `[+]ping ok
 [+]log ok
-[+]poststarthook/generic-apiserver-start-informers ok
-[+]informer-sync ok
```

Comment:
ok, the solution is not perfect :P 
We should try to not loose this health checks :P 
I don't think all are required however `[+]informer-sync ok` seems important.

Reply:
Ok. It seems that we need to start an informer for the apiserver separately. I will update it

Reply:
We might want to register our own probes.

Reply:
OK. I will study how to complete